gdx/src/com/badlogic/gdx/AbstractGraphics.java

```diff
@@ -1,19 +1,29 @@
 
 package com.badlogic.gdx;
 
+/** Abstract implementation of the Graphics interface, providing common methods for graphics-related functionalities such as time
+ * and density calculations. */
 public abstract class AbstractGraphics implements Graphics {
 
+	/** Delegates to {@link #getDeltaTime()} for the actual time calculation.
```

Comment:
If a method already has javadocs on the super method, it doesn't need new javadocs.